src/components/BorrowerProfile/HeroBackground.vue

```diff
@@ -8,7 +8,7 @@
 	>
 		<kv-contentful-img
 			class="tw-w-full tw-h-full tw-object-cover"
-			:contentful-src="contentfulSrc"
+			:contentful-src="statusCard ? optimizedImageUrl : contentfulSrc"
```

Comment:
This implementation won't work because `kv-contentful-img` already constructs its own url with query parameters. For this component, you can just pass in the correct numbers for the props of `height`, `width` and `source-sizes`